### ISSUE TITLE
Private access right may not work

### ISSUE THREAD

**Issue by YunzhongTianjing** [hongyangAndroid/Android-HyViewInject#2]
By reading the source code,I doubt that the view modified by private-modifier can't be injected,isn't it?

**Comment by hongyangAndroid** [comment]
Yes , you are right . I guess any inject framework based on annotation processing  can not support private-modifier  .
